src/panel/direction/DirectionPanel.jsx

```diff
@@ -332,6 +332,7 @@ export default class DirectionPanel extends React.Component {
           className="direction_panel"
         >
           {form}
+          <div id="itinerary_autocomplete_suggestions" />
```

Comment:
The suggestions list remain visible after the last field (that triggers the directions search) is submitted with Enter key:

<img src="https://user-images.githubusercontent.com/4726554/82063091-5661d100-96cb-11ea-9116-cea9562426e1.png" width="250px">

Reply:
This bug turns out to be specific to Firefox.

Reply:
thanks, it should be fixed now